pkg/api/mlflow/dao/repositories/helpers.go

```diff
@@ -30,3 +32,32 @@ func makeParamConflictPlaceholdersAndValues(params []models.Param) (string, []in
 	}
 	return placeholders, valuesArray
 }
+
+// BuildJsonCondition creates sql and values for where condition to select items having the specified map of json paths
+// and values in the given json column. Json path is expressed as "key" or "outerkey.nestedKey".
+func BuildJsonCondition(
+	dialector string,
+	jsonColumnName string,
+	jsonPathValueMap map[string]string,
+) (sql string, args []any) {
+	if len(jsonPathValueMap) == 0 {
+		return
+	}
+	var conditionTemplate string
+	switch dialector {
+	case postgres.Dialector{}.Name():
+		conditionTemplate = "%s#>>? = ?"
+		for k, v := range jsonPathValueMap {
+			path := strings.ReplaceAll(k, ".", ",")
+			args = append(args, "{"+path+"}", v)
+		}
+	default:
+		conditionTemplate = "%s->>? = ?"
+		for k, v := range jsonPathValueMap {
+			args = append(args, k, v)
+		}
+	}
+	conditionTemplate = fmt.Sprintf(conditionTemplate, jsonColumnName)
+	sql = strings.Repeat(conditionTemplate+" AND ", len(jsonPathValueMap)-1) + conditionTemplate
+	return
```

Comment:
ohh, naming parameters always looks strange. can we please just return `sql` and `args` explicitly?

Reply:
well i thought this was a case where the names helped for the return vals in the signature -- but I will make explicit return here